### PR TITLE
get active user

### PR DESCRIPTION
**Pull request by dev-mohmmed-alshorafa**
#102

### CODE REVIEW

**Review by Saeed99Madi — APPROVED** [review]

**Review by AseelL — APPROVED** [review]

File: client/src/helpers/SendIsActive.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { doc, setDoc, serverTimestamp } from 'firebase/firestore';
+import IUser from '../Interfaces/post/IUser';
+import { db } from './Firebase.config';
+
+async function SendIsActive(user: IUser, state: boolean) {
```
Comment (Amoodaa) [inline client/src/helpers/SendIsActive.ts:5]: rename to setActiveUser

File: client/src/helpers/deleteOffLine.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { doc, deleteDoc } from 'firebase/firestore';
+import { db } from './Firebase.config';
+
+async function deleteOffLine(userId: number | undefined) {
```
Comment (Amoodaa) [inline client/src/helpers/deleteOffLine.ts:4]: rename to `unsetActiveUser`

File: client/src/helpers/getWhoActive.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { collection, onSnapshot, query, orderBy } from 'firebase/firestore';
+import { db } from './Firebase.config';
+
+// eslint-disable-next-line promise/prefer-await-to-callbacks
+function getWhoActive(callback: Function) {
```
Comment (Amoodaa) [inline client/src/helpers/getWhoActive.ts:5]: rename to onActiveUsersChange

File: client/src/pages/Emergency.tsx [hunk 4]
```diff
@@ -33,9 +31,12 @@ const Emergency = () => {
...
         Available Emergency Doctors
       </p>
-      {doctors.map((doctor: IUser) => (
-        <Doctor key={doctor.id} user={doctor} />
-      ))}
+
+      {user ? (
+        doctors.map((doctor: IUser) => <Doctor key={doctor.id} user={doctor} />)
+      ) : (
+        <h1>you have to signIn pro!</h1>
```
Comment (Amoodaa) [inline client/src/pages/Emergency.tsx:38]: hm

**Review by aaamra — APPROVED** [review]

**Pull request merged**